PURL-TYPES.rst

```diff
@@ -402,6 +402,7 @@ maven
       pkg:maven/org.apache.xmlgraphics/batik-anim@1.9.1?type=zip&classifier=dist
       pkg:maven/net.sf.jacob-projec/jacob@1.14.3?classifier=x86&type=dll
       pkg:maven/net.sf.jacob-projec/jacob@1.14.3?classifier=x64&type=dll
+      pkg:maven/groovy/groovy@1.0?repository_url=https://repo1.maven.org/maven2
```

Comment:
Could we have another repo for the example? I do not want to promote that adding the default is a thing. 
May be instead something like: `pkg:maven/groovy/groovy@1.0?repository_url=https://maven.google.com`

Reply:
Good call. Addressed in [c131917](https://github.com/package-url/purl-spec/pull/322/commits/c131917b41bd6c492a216a31982cb8a5ce09b586)